mapadroid/route/RouteManagerBase.py

```diff
@@ -751,8 +756,7 @@ def _other_worker_closer_to_prioq(self, prioqcoord, origin):
         temp_distance = distance_worker
 
         for worker in self._routepool.keys():
-            if worker == origin or self._routepool[worker].has_prio_event \
-                    or self._routepool[origin].last_round_prio_event:
```

Comment:
I am not sure why this check exists in this loop. Worst case it should be near the top of the method.. because.. why do all of the work in this method when it's False? :)

Instead of moving it, I've chosen to just remove it. I don't see what it buys... it can only possibly be true here for 'starve_route'.